src/couch_replicator_scheduler.erl

```diff
@@ -207,14 +207,17 @@ not_recently_crashed(#job{} = Job) ->
     case crash_history(Job) of
         [] ->
             true;
-        [{crashed, When} | _] ->
+        [{{crashed, _Reason}, When} | _] = Crashes ->
+            BackoffExp = erlang:min(1 bsl length(Crashes), ?MAX_BACKOFF_EXPONENT),
```

Comment:
`?MAX_BACKOFF_EXPONENT` is not the exponent though, `length(Crashes)` is the exponent.